ui/src/app/store/pod/utils.ts

```diff
@@ -29,3 +29,15 @@ export const getPodNumaID = (machine: Machine, pod: Pod): number | null => {
   }
   return null;
 };
+
+export const getCoreIndices = (
```

Comment:
Might be good to have a few comments here about what this is doing...

Reply:
oh yeah good point